### DISCUSSION TITLE
[Feature Request] Add type Week and Month to Input

### DISCUSSION THREAD

**Discussion by LucasLeone** [heroui-inc/heroui#4707] [Feature requests]
### Is your feature request related to a problem? Please describe.

I'm using input with type week and month but they aren't available in all browsers. I think is a good idea to add it.

### Describe the solution you'd like

As a user, I need Input can handle Week and Month picker correctly in all browsers.

### Describe alternatives you've considered

Maybe create a new component.

**Comment by linear** [comment]
<p><a href="https://linear.app/nextui-inc/issue/ENG-1791/[feature-request]-add-type-week-and-month-to-input">ENG-1791 [Feature Request] Add type Week and Month to Input</a></p>
